network-api/networkapi/wagtailpages/factory/libraries/research_hub/detail_page.py

```diff
@@ -38,22 +38,24 @@ def collaborators(self):
         return "; ".join(names)
 
     research_authors = factory.RelatedFactoryList(
-        factory="networkapi.wagtailpages.factory.research_hub.relations.ResearchAuthorRelationFactory",
+        factory="networkapi.wagtailpages.factory.libraries.research_hub.relations.ResearchAuthorRelationFactory",
         factory_related_name="research_detail_page",
         size=1,
     )
 
     related_topics = factory.RelatedFactoryList(
         factory=(
-            "networkapi.wagtailpages.factory.research_hub.relations.ResearchDetailPageResearchTopicRelationFactory"
+            "networkapi.wagtailpages.factory.libraries.research_hub"
```

Comment:
Had to separate this into two lines, or else it would go over the linters char limit.